CIP-????/README.md

```diff
@@ -0,0 +1,172 @@
+
+
+On rare occasion a threat may be identified that requires responsible reporting procedures in order to prevent exposing an active vulnerability to bad actors who may aim to exploit the vulnerability. In the event responsible reporting is required, notify the appropriate group or entity with the responsibility and resources to mitigate such threat in a confidential manner.
+
+Assumption: There are no known software back doors. Unknown software vulnerabilities may be discovered.
+
+### #1: Unknown software vulnerability affecting governance is discovered.
+
+Recommendation: The person discovering the vulnerability should document the circumstances and details in which the vulnerability is present and how to reproduce the vulnerability. Then report the vulnerability in a confidential manner using a secure method to the developer team with the resources to correct the vulnerability.
+
+### #2: Emergency group to support the disaster recovery plan.
+  -attacks on persons if known
+  -we don’t want attackers know who that group is?
+
+Recommendation:
+
+### #3: Emergency change that needs to be done quickly.
+  -a nuclear option
+  -change requires “just do it” 
+  -a security that fix must be implemented but CANNOT be communicated publicly
+
+Recommendation:
+
+### #4: Operational Security (OPSEC) of the committee.
+  -physical security
+  -digital security
+
+Recommendation:
+
+### #5: Committee keys lost or stolen or committee keys sold to a bad actor.
+
+Recommendation:
+
+### #6: DRep keys lost or stolen or DRep keys sold to a bad actor.
+
+Recommendation:
+
+### #7: Lack of governance for an extended period of time.
+  -Uncontrolled hard forks by SPOs
+  -Risk of no confidence
+
+Recommendation:
+
+### #8: Lack of trust in voter tooling.
+
+Recommendation:
+
+### #9: Trust in the wallets.
+  -wallet voting attack vector
+  -hardware wallet support
+
+Recommendation:
+
+
+### #10: Corruption - in any form, any number of actors.
+  -Code of Conduct mitigation
+  -on-chain mitigation
+
+
+Recommendation:
+
+### #11: Collusion - the bad version of collaboration to do something illegal or undesirable.
+
+Recommendation:
+
+### #12: Vote buying - Ada kickbacks.
+
+Recommendation:
+
+### #13: IGCO - initial governance coin offering
+  -offering voters a token of unknown value by a representative to acquire delegators
+  -including tokens other than Ada
+
+Recommendation:
+
+### #14: Powerful Voter Collusion
+  -demotivates smaller bag holders
+  -cause a loss of participation
+  -too many voters abstain
+  -minority controls the outcome
+  -can push through any proposal
+
+Recommendation:
+
+### #15: Persistence - Removal of proposals by attackers.
+
+Recommendation:
+
+### #16: Risk of Proposal overload - Risk of DDOS.
+
+Recommendation:
+
+### #17: Excessive actions.
+  -Drawing too much money
+  -Changing parameters that break the chain (block size=0 for example)
+  -overlapping actions and duplicate actions
+
+Recommendation:
+
```

Comment:
- Intentional attempts to submit parameter change proposals that are deemed detrimental to the health of the network should result in a loss of collateral. As mentioned for line 118, collateral should be required to submit proposals.
- overlapping/duplicate actions should be mostly curbed due to collateral requirement. If 1/1 duplicate is successfully proposed, the enforcement of a penalty from collateral as a percentage might be appropriate.